src/components/Messages/MessagesListItem.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  ListItemButton,
+  ListItemAvatar,
+  Avatar,
+  ListItemText,
+  Stack,
+} from "@mui/material";
+
+const styles = {
+  avatar: { margin: "auto" },
+};
+
+export type LatestMessageDetails = {
+  displayName: string;
+  username: string;
+  textContent: string;
+  timestamp: string;
+};
+
+type MessageListItemProps = {
+  latestMessageDetails: LatestMessageDetails;
+};
+
+const MessagesListItem = ({ latestMessageDetails }: MessageListItemProps) => {
+  return (
+    <ListItemButton>
+      <Stack direction="row">
+        <ListItemAvatar sx={styles.avatar}>
+          <Avatar />
+        </ListItemAvatar>
+        <ListItemText
+          primary={`${latestMessageDetails.displayName} @${latestMessageDetails.username} - ${latestMessageDetails.timestamp}`}
```

Comment:
This might be something we fix later, but I think we should figure out a way for us to ensure that the name does not spill over to a second line. Right now the timestamp spills over, which will be fixed at a later date, but if we end up creating a long display name this is something that should be cut-off by maybe adding `"..."` or something at the end of the string. 

That way the Items don't lose that clean look.

Reply:
I'm also not sure how we can do this with the `primary` attribute, but is there any way we can change the color of the username so that it doesn't mesh with the display name?